site/e2e/tests/auditLogs.spec.ts

```diff
@@ -0,0 +1,69 @@
+import { expect, test } from "@playwright/test";
+import {
+  createTemplate,
+  createWorkspace,
+  requiresEnterpriseLicense,
+} from "../helpers";
+import { beforeCoderTest } from "../hooks";
+
+test.beforeEach(async ({ page }) => await beforeCoderTest(page));
+
+test("inspecting and filtering audit logs", async ({ page }) => {
+  requiresEnterpriseLicense();
+
+  const userName = "admin";
+  // Do some stuff that should show up in the audit logs
+  const templateName = await createTemplate(page);
+  const workspaceName = await createWorkspace(page, templateName);
+
+  // Go to the audit history
+  await page.goto("/audit");
+
+  // Make sure those things we did all actually show up
+  await expect(page.getByText(`${userName} logged in`)).toBeVisible();
+  await expect(
+    page.getByText(`${userName} created template ${templateName}`),
+  ).toBeVisible();
+  await expect(
+    page.getByText(`${userName} created workspace ${workspaceName}`),
+  ).toBeVisible();
+  await expect(
+    page.getByText(`${userName} started workspace ${workspaceName}`),
+  ).toBeVisible();
+
+  // Make sure we can inspect the details of the log item
+  const createdWorkspace = page.locator(".MuiTableRow-root", {
+    hasText: `${userName} created workspace ${workspaceName}`,
+  });
+  await createdWorkspace.getByLabel("open-dropdown").click();
+  await expect(
+    createdWorkspace.getByText(`automatic_updates: "never"`),
+  ).toBeVisible();
+  await expect(
+    createdWorkspace.getByText(`name: "${workspaceName}"`),
+  ).toBeVisible();
+
+  const startedWorkspaceMessage = `${userName} started workspace ${workspaceName}`;
+  const loginMessage = `${userName} logged in`;
+
+  // Filter by resource type
+  await page.getByText("All resource types").click();
```

Comment:
Not fully sure what element this is, but could the selector be updated to have a `link`/`button` role, since whatever it is, it looks like something that can be interacted with?

Reply:
there are multiple buttons on the page though. those roles would be ambiguous.